### PR TITLE
WX-1122 Use legacy AppInsights to get better control over logging

### PR DESCRIPTION
**Pull request by jgainerdewar**
This reverts #7143 and adds an older type of Application Insights log integration. The newer version didn't give us enough control over what logs were sent. Specifically, the fact that Cromwell uses a per-workflow logger (to create user-accessible log files) in addition to the root logger resulted in all workflow-related messages being duplicated in Application Insights.

### CODE REVIEW

File: server/src/main/resources/logback.xml [hunk 1]
```diff
@@ -94,11 +94,14 @@
         </filter>
     </appender>
 
+    <appender name="AppInsightsAppender" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender" />
```
Comment (jgainerdewar) [inline server/src/main/resources/logback.xml:97]: Given that Sentry is in here is seemed appropriate to include this.

**Comment by jgainerdewar** [comment]
Got some build issues here, will fix.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -12,7 +12,7 @@ object Dependencies {
   // https://github.com/sbt/sbt/issues/4531
   private val azureStorageBlobNioV = "12.0.0-beta.19"
   private val azureIdentitySdkV = "1.9.0-beta.2"
-  private val azureAppInsightsV = "3.4.12"
+  private val azureAppInsightsLogbackV = "2.6.4"
```
Comment (THWiseman) [inline project/Dependencies.scala:15]: Do we want to leave a comment here warning the future upgrade-hungry developer about consequences of bumping this version?

**Review by THWiseman — APPROVED** [review]
LGTM. Optional request to leave one more comment behind next to the version pin.

File: project/Dependencies.scala (hunk 2 above)
Comment (jgainerdewar) [reply project/Dependencies.scala:15]: Yeah, that's a good idea.

**Review by aednichols — APPROVED** [review]

**Pull request merged**